core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -1288,15 +1297,12 @@ object Pull extends PullLowPriority {
     go(initScope, None, initFk, new OuterRun(init), stream)
   }
 
+  @deprecated("use the extension method", "3.3.1")
```

Comment:
The `tlBaseVersion` is already 3.4.0 (b/c of the Scala 3.2 bump)

```suggestion
  @deprecated("use the extension method", "3.4.0")
```